app/services/attachment_generator.rb

```diff
@@ -0,0 +1,45 @@
+class AttachmentGenerator
+  MAX_ATTACHMENTS_SIZE = 10_000_000 # 10MB in bytes. AWS SES limitation
```

Comment:
[According to the SES docs](https://aws.amazon.com/ses/faqs/#Limits_and_Restrictions), the 10MB limit applies to the whole email, not just attachments.

Reply:
ooh good spot! will adjust this to cater for that. Thanks, Jenny :)